### PR TITLE
cli: truncate groups in get clusterrolebindings

### PR DESCRIPTION
**Pull request by mfojtik**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1539415

### CODE REVIEW

File: pkg/oc/cli/describe/printer.go [hunk 1]
```diff
@@ -858,6 +858,14 @@ func printRoleList(list *authorizationapi.RoleList, w io.Writer, opts kprinters.
 	return nil
 }
 
+func truncatedList(list []string, maxLength int) string {
+	result := strings.Join(list, ", ")
+	if len(list) > maxLength {
+		result = strings.Join(list[0:maxLength], ", ") + fmt.Sprintf(" (%d more)", len(list)-maxLength)
```
Comment (tnozicka) [inline pkg/oc/cli/describe/printer.go]: I find this more readable and efficient
`fmt.Sprintf("%s (%d more)", strings.Join(list[0:maxLength], ", "), len(list)-maxLength)`

**Comment by tnozicka** [comment]
nit, lgtm otherwise

File: pkg/oc/cli/describe/printer.go (hunk 1 above)
Comment (mfojtik) [reply pkg/oc/cli/describe/printer.go]: yeah, it can be optimized even more ;-) fixed.

**Pull request merged**